Bender.yml

```diff
@@ -0,0 +1,110 @@
+  apb_node:               { git: "git@github.com:pulp-platform/apb_node.git", version: 0.1.1 }
+  apb_interrupt_cntrl:    { git: "git@github.com:pulp-platform/apb_interrupt_cntrl.git", version: 0.1.1 }
+  axi:                    { git: "git@github.com:pulp-platform/axi.git", version: 0.24.1 }
+  # axi_node:               { git: "git@github.com:pulp-platform/axi_node.git", version: 1.1.4 } # deprecated, replaced by axi_xbar (in axi repo)
+  axi_slice:              { git: "git@github.com:pulp-platform/axi_slice.git", version: 1.1.4 } # deprecated, replaced by axi_cut (in axi repo)
+  axi_slice_dc:           { git: "git@github.com:pulp-platform/axi_slice_dc.git", version: 1.1.3 } # deprecated, replaced by axi_cdc (in axi repo)
+  timer_unit:             { git: "git@github.com:pulp-platform/timer_unit.git", version: 1.0.2 }
+  common_cells:           { git: "git@github.com:pulp-platform/common_cells.git", version: 1.20.0 }
+  cluster_interconnect:   { git: "git@github.com:pulp-platform/cluster_interconnect.git", version: 1.1.1 }
+  fpnew:                  { git: "git@github.com:pulp-platform/fpnew.git", version: 0.6.5 }
+  jtag_pulp:              { git: "git@github.com:pulp-platform/jtag_pulp.git", rev: "v0.1" }
+  riscv:                  { git: "git@github.com:micprog/cv32e40p.git", rev: "pulpissimo-v3.4.1"} # To be updated to openhwgroup repository
+  ibex:                   { git: "git@github.com:lowRISC/ibex.git", rev: "b148f3af51066ac3537c7c6eb4acbc17004bf868" }
+  scm:                    { git: "git@github.com:pulp-platform/scm.git", version: 1.0.1}
+  generic_FLL:            { git: "git@github.com:pulp-platform/generic_FLL.git", rev: "1c92dc73a940392182fd4cb7b86f35649b349595" }
+  tech_cells_generic:     { git: "git@github.com:pulp-platform/tech_cells_generic.git", rev: "e1655889cf947b0f39c3f4e95410487d3505a771" }
+  udma_core:              { git: "git@github.com:pulp-platform/udma_core.git", version: 1.0.2 }
+  udma_uart:              { git: "git@github.com:pulp-platform/udma_uart.git", version: 1.0.1 }
+  udma_i2c:               { git: "git@github.com:pulp-platform/udma_i2c.git", version: 1.0.0 }
+  udma_i2s:               { git: "git@github.com:pulp-platform/udma_i2s.git", version: 1.1.1 }
+  udma_qspi:              { git: "git@github.com:pulp-platform/udma_qspi.git", version: 1.0.3 }
+  udma_sdio:              { git: "git@github.com:pulp-platform/udma_sdio.git", version: 1.1.0 }
+  udma_camera:            { git: "git@github.com:pulp-platform/udma_camera.git", version: 1.1.1 }
+  udma_filter:            { git: "git@github.com:pulp-platform/udma_filter.git", version: 1.0.2 }
+  udma_external_per:      { git: "git@github.com:pulp-platform/udma_external_per.git", version: 1.0.3 }
+  hwpe-mac-engine:        { git: "git@github.com:micprog/hwpe-mac-engine.git", rev: "v1.2-bender" } # To be updated with pulp-platform dependency
```

Comment:
And this one? We should close open PR (where they are under our control) before merging to master...

Reply:
The HWPE repositories have significant updates within, the versions referenced in pulp are 2-3 years old if I'm not mistaken. I believe @FrancescoConti is working on updating this, I only added a `Bender.yml` file to the versions indicated by pulp. I did not investigate the current master and the impact the changes to these dependencies may have, but assumed that it may break compatibility. A temporary solution could be to add a branch to this repository as well, but [hwpe-mac-engine](https://github.com/pulp-platform/hwpe-mac-engine) also depends on [hwpe-ctrl](https://github.com/pulp-platform/hwpe-ctrl) and [hwpe-stream](https://github.com/pulp-platform/hwpe-stream), so these would require similar treatment.